geopandas_view/view.py

```diff
@@ -422,7 +426,7 @@ def _choropleth(
         bins.insert(0, gdf[column].min())
 
     choro = folium.Choropleth(
-        gdf,
+        gdf.__geo_interface__,
```

Comment:
This was added for the `test_no_crs` right? do you think this should be updated in folium cause updating https://github.com/python-visualization/folium/blob/15f9ceca6c89a718c3819aefd67efa0fc93b896a/folium/features.py#L541 to `data.crs = 'EPSG:4326'` seems to fix this and there would no need to use `__geo_interface__` here. I could just raise a PR over there

Reply:
> This was added for the test_no_crs right?

Yes. But for naive geometry, we don't want to assume it is 4326 because it gets automatically reprojected to 3857. We should plot those geometries as they are. I am also turning off background tiles in that case.

Reply:
Got it ! was not aware of the reprojection but I observed that changing this in folium turns off the background tiles as well and the tests still pass with just `gdf` but I guess it does not make much of a difference